### PR TITLE
feat(sdk-coin-aca): add sdk-coin-aca module

### PR DESCRIPTION
**Pull request by alia-bitgo**
TICKET: [BG-52862](https://bitgoinc.atlassian.net/browse/BG-52862?atlOrigin=eyJpIjoiNGNiM2E4MDczZjhhNDA3Mzg4MmY1NGRlMTBjMGQ5NDYiLCJwIjoiaiJ9)

### CODE REVIEW

**Review by lcovar — DISMISSED** [review]

File: modules/sdk-coin-aca/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "compilerOptions": {
+    "outDir": "./dist",
+    "rootDir": "./",
+    "strictPropertyInitialization": false,
+    "esModuleInterop": true,
+    "typeRoots": ["../../types", "./node_modules/@types", "../../node_modules/@types"]
+  },
+  "include": ["src/**/*", "test/**/*"],
+  "exclude": ["node_modules"],
+  "references": [
```
Comment (byhow) [inline modules/sdk-coin-aca/tsconfig.json:12]: need `"path": "../statics"`

File: modules/sdk-coin-aca/package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "access": "public"
+  },
+  "nyc": {
+    "extension": [
+      ".ts"
+    ]
+  },
+  "dependencies": {
+    "@bitgo/sdk-core": "^1.1.0",
+    "@bitgo/statics": "^7.0.0"
```
Comment (byhow) [inline modules/sdk-coin-aca/package.json:45]: Will we be adding this coin to stats in a separate pr?

**Review by byhow — COMMENTED** [review]
overall LGTM

File: modules/sdk-coin-aca/package.json (hunk 2 above)
Comment (alia-bitgo) [reply modules/sdk-coin-aca/package.json:45]: aca already added to statics: https://github.com/BitGo/BitGoJS/pull/2550

File: modules/sdk-coin-aca/tsconfig.json (hunk 1 above)
Comment (alia-bitgo) [reply modules/sdk-coin-aca/tsconfig.json:12]: just added it in

**Review by byhow — APPROVED** [review]

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**